third_party/memfault/port/src/memfault_chunk_collector.c

```diff
@@ -0,0 +1,74 @@
+// Logging Memfault chunks to datalogging can only happen in normal FW
```

Comment:
exclude file from the build in wscript if !MEMFAULT?

Reply:
I'm not sure how to do that - but it seems like it already is automatically? It doesn't compile this file when I configure for e.g. `snowy_bb2`